### PR TITLE
Commerce 4953 Shipping Method rates should be defined in the channel's currency and not the VI's currency

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Review by riccardo-alberti — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-shipping-engine-fixed-web/src/main/java/com/liferay/commerce/shipping/engine/fixed/web/internal/display/context/BaseCommerceShippingFixedOptionDisplayContext.java [hunk 1]
```diff
@@ -37,24 +39,35 @@
 public class BaseCommerceShippingFixedOptionDisplayContext {
 
 	public BaseCommerceShippingFixedOptionDisplayContext(
+		CommerceChannelLocalService commerceChannelLocalService,
 		CommerceCurrencyLocalService commerceCurrencyLocalService,
 		CommerceShippingMethodService commerceShippingMethodService,
 		RenderRequest renderRequest, RenderResponse renderResponse) {
 
+		this.commerceChannelLocalService = commerceChannelLocalService;
```
[the remark this answers is not in the document]
Comment (luca-pellizzon) [reply modules/apps/commerce/commerce-shipping-engine-fixed-web/src/main/java/com/liferay/commerce/shipping/engine/fixed/web/internal/display/context/BaseCommerceShippingFixedOptionDisplayContext.java:47]: Here we are using the service only for backend purposes, no user input is managed, also the method is only available in local service.

**Review by dsitu — APPROVED** [review]

**Review by riccardo-alberti — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**